app/Http/Controllers/Item/ItemController.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use App\Http\Controllers\Item\ItemRepository;
+use App\Item;
+use app\Http\Requests\CreateItemRequest;
+
+class ItemController extends Controller
+{
+    private $itemRepository;
+
+    public function __construct(ItemRepository $itemRepository)
+    {
+        $this->itemRepository = $itemRepository;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // TO-DO
+        // Ask fro help from Jeseph of how to use the config values on pagination
+        // get the pagination number or a default
+        $items = $this->$itemRepository ?? 10;
+        return view('items' . config('settings.pagination.small') . '.index')->with(['$items' => $items]);
```

Comment:
@okellogabrielinnocent Not sure what's going on here. Are you showing a different view based on the pagination number?

Reply:
I read on how to paginate this to the view
and as you can see in my comment I'll need your help on how to use the `config('settings.pagination.small')` with pagination rather than the `paginate($number)`